### PR TITLE
Search movies

### PR DESCRIPTION
**Pull request by WoytekMig**
changed component from searching people to search either people or movies. Added loading screen during fetching data from api and changed title during loading data :)

### CODE REVIEW

File: src/features/MoviesBrowser/SearchResult/index.js [hunk 1]
```diff
@@ -46,24 +39,26 @@ const SearchResult = () => {
...
-            : `Search result for "${queryParam || ""}" (${
+          isLoading
+            ? `Search result for "${queryParam || ""}" (${
                 data?.results.length || 0
               })`
+            : isError || data.results.length === 0 || !data
+            ? `Sorry, there are no results for "${queryParam || ""}"`
+            : `Search result for "${queryParam || ""}" (${data.total_results})`
         }
       />
```
Comment (KrystianZybura) [inline src/features/MoviesBrowser/SearchResult/index.js:77]: Isn't it a bit too complicated?

**Review by KrystianZybura — CHANGES_REQUESTED** [review]
Nice job! Sadly there is an error, when we try to search for Movies way too fast meanwhile loading due to missing data for MovieTile. It occurs only when, we use the navigation from People to Movies and using search during the loading phase.
image


Other than that, the search doesn't work on Profile and Movie Page as we discussed earlier, it should be fixed before merge.

**Review by Arcziwald — APPROVED** [review]
Good job :-)

**Review by KrystianZybura — APPROVED** [review]
Nice job, works well 👌

**Pull request merged**